data/resources.json

```diff
@@ -8,10 +8,12 @@
   "followOur": "Follow our",
   "followUs": "Follow us",
   "emailUs": "Email us",
-  "blog": "Blog",
   "partnershipResources": "Partnership Resources",
   "eventResources": "Event Resources",
   "buyTicket": "Buy ticket",
+  "footer": {
+    "blog": "Blog"
```

Comment:
Nope, there is different property, as people can rename them as they want.
Btw, please add scope to your variables